torch_np/_util.py

```diff
@@ -193,7 +196,6 @@ def _coerce_to_tensor(obj, dtype=None, copy=False, ndmin=0):
     """
     if isinstance(obj, torch.Tensor):
         tensor = obj
-        base = None
     else:
         tensor = torch.as_tensor(obj)
         base = None
```

Comment:
also this one, now that we're at it. Here or in any other PR really.

Reply:
Yup, included in the next one.